sites/website/README.md

```diff
@@ -2,24 +2,28 @@
 
 This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
 
-### Installation
+## Installation
 
-```
-$ yarn
+```sh
+yarn
 ```
 
-### Local Development
+## Local Development
 
-```
-$ yarn start
+```sh
+yarn start
 ```
 
-This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server. Note that the homepage won't be displayed when developing locally. See the section "Local Serve" below to test the site in a complete build state.
 
```

Comment:
I have been able to view the homepage when developing locally by clicking on the FAST logo in the upper left hand corner of the docs site, but clicking on the "Home" icon in the navbar routes me to fast.design.

Reply:
Can you double-check that it's not taking you to the live production homepage instead of the local development version?

The prebuild step builds the `sites/fast-website` project first, then places its build output into the `static` directory before running Docusaurus. Docusaurus takes everything in the `static` directory as-is and includes it in the final `build` directory. On `2.0.0-beta.1`, it allowed this baked-in homepage to display alongside the Docusaurus output while using `yarn start`, but that doesn't seem to happen for me anymore on beta 14.

Reply:
I'm able to do this?  🤷‍♀️

![Screen Shot 2022-01-26 at 11 48 27 AM](https://user-images.githubusercontent.com/35910788/151237027-638e4fb1-ed70-4744-a996-d9c96bc97bb4.png)

Reply:
Here's my result, on both MacOS and Windows/WSL2:

<img width="1302" alt="Screen Shot 2022-01-26 at 12 21 37 PM" src="https://user-images.githubusercontent.com/863023/151245459-a91c9d59-52e5-4768-9480-40914659a8df.png">